packages/models-library/src/models_library/api_schemas_webserver/users_preferences.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, TypeAlias
+
+from pydantic import BaseModel, Field
+
+from ..user_preferences import PreferenceIdentifier
+
+
+class FrontendUserPreference(BaseModel):
+    default_value: Any = Field(
+        default=..., alias="defaultValue", description="used by the frontend"
+    )
+    value: Any = Field(default=..., description="preference value")
+
+    class Config:
+        allow_population_by_field_name = True
+
+
+FrontendUserPreferencesGet: TypeAlias = dict[
```

Comment:
IMO this is very misleading. With a response payload ending in `Get`, I  would have expected an object but in reality you are returning a list "in disguise" .  

In reality what you want is a `List`, `Get` and `Patch` operations
-  `list_user_perferences`: `GET preferences` -> `Envelope[list[FrontendUserPreference]]
-  `get_user_preferences`: `GET preferences/{preference_id}` -> `FrontendUserPreference`
- `update_user_preferences`: `PATCH preferences/{preference_id}`,  -> `FrontendUserPreference`